docs/sources/tempo/configuration/_index.md

```diff
@@ -1559,6 +1559,10 @@ overrides:
       # Per-user compaction window. If this value is set to 0 (default),
       # then block_retention in the compactor configuration is used.
       [compaction_window: <duration> | default = 0s]
+      # Allow compaction to be deactivated on a per-tenant basis. Default value
+      # is 0 (compaction active). Useful to perform operations on the backend
+      # that require compaction to be disabled for a period of time.
+      [compaction_disabled: <bool> | default = 0]
```

Comment:
@zalegrala I added an entry for compaction_disabled based on the code in your PR. Is this correct?

Reply:
Default value is `false`.

Reply:
```suggestion
      # Allow compaction to be deactivated on a per-tenant basis. Default value
      # is 0 (compaction active). Useful to perform operations on the backend
      # that require compaction to be disabled for a period of time.
      [compaction_disabled: <bool> | default = false]
```